projectq/meta/_compute.py

```diff
@@ -87,6 +87,21 @@ def _add_uncompute_tag(self, cmd):
         cmd.tags.append(UncomputeTag())
         return cmd
 
+    def _send_rewritten_command(self, cmd, rewrite_id_map):
```

Comment:
(The reason I only moved one branch is because I want to keep the PRs small. I intend to extract several other private functions out of this large method.)